apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -141,6 +375,60 @@ static void tc_libc_math_fdim(void)
 }
 
 /**
+ * @fn                   :tc_libc_math_fdimf
+ * @brief                :Returns the positive  difference max(x-y,0),  between thier arguments
+ * @Scenario             :Returns the positive  difference max(x-y,0),  between their arguments
+ * API's covered         :fdimf
+ * Preconditions         :None
+ * Postconditions        :None
+ * @return               :void
+ */
+static void tc_libc_math_fdimf(void)
+{
+	const float in_val1[10] = { ZERO, -ZERO, VAL1, -VAL1, VAL2, -VAL2, INFINITY, -INFINITY, NAN, VAL1 };
+	const float in_val2[10] = { -INFINITY, INFINITY, -VAL2, VAL2, -VAL1, VAL1, -ZERO, ZERO, NAN, NAN };
+	const float sol_val[10] = { INFINITY, ZERO, 6755399441055744.0, ZERO, 6755399441055744.0, ZERO, INFINITY, ZERO, NAN, NAN };
+	float ret_val[10];
+	int fdimf_idx;
+
+	for (fdimf_idx = 0; fdimf_idx < 9; fdimf_idx++) {
```

Comment:
1. here is 9. but above size is 10. is this right? 
2. how about modify these kind codes?
    1. remove fixed size at definition
    2. use sizeof instead of fixed value